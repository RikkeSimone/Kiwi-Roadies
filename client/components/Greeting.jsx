import React from 'react'
import { connect } from 'react-redux'

class Greeting extends React.Component {
  render () {
    return (
      <>
        { this.props.name &&
           <div className="greeting-text">
             <h2>hey, {this.props.name}!</h2>
             <p>Tell us where you would like to go!</p>
           </div>
        }
      </>
    )
  }
}

const mapStateToProps = state => {
  return {
    name: state.name
  }
}

export default connect(mapStateToProps)(Greeting)
