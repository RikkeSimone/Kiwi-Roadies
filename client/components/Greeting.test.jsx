import React from 'react'
import { renderWithRedux } from '../testing/utils'
import '@testing-library/jest-dom'
import 'regenerator-runtime/runtime'

import Greeting from './Greeting'

test('Greeting displays welcoming message', async () => {
  const { asFragment } = renderWithRedux(<Greeting />)
  expect(asFragment()).toMatchSnapshot()
})
