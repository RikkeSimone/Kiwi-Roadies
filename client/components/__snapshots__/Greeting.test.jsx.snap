// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting displays welcoming message 1`] = `<DocumentFragment />`;
